scripts/dts/gen_driver_kconfig_dts.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Kumar Gala <galak@kernel.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+import re
+from collections import defaultdict
+
+import yaml
+try:
+    # Use the C LibYAML parser if available, rather than the Python parser.
+    # This makes e.g. gen_defines.py more than twice as fast.
```

Comment:
this message doesn't seem relevant here?

Reply:
will fix, copy/pasted from gen_defines.py ;)